identity/app/utils/Skin.scala

```diff
@@ -0,0 +1,20 @@
+package utils
+
+object Skin {
```

Comment:
do we have anything similar that could fulfil this purpose already? I don't know off the top of my head, but it's a shame if we don't

Reply:
and is this necessary if we only expect to have 2 skins?

Reply:
Can't vouch for the necessity of it but it felt cleaner than passing a string around all the objects this touches